src/internal/id.rs

```diff
@@ -114,6 +114,7 @@ impl ArenaId for ClauseId {
         Self(unsafe { NonZeroU32::new_unchecked((x + 1).try_into().expect("clause id too big")) })
     }
 
+    #[inline]
```

Comment:
Wouldn't this be inlined automatically? It seems trivial if it could and otherwise it might not benefit?